content/docs/guides/ha_scale/perf-scale.md

```diff
@@ -8,7 +8,7 @@ Open Service Mesh aims to create a high-performance service mesh while providing
 
 ## Testing environment
 
-This test is performed on a Kubernetes cluster provisioned and managed by [Microsoft Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service/). The cluster has 100 Standard_DS2_v2 nodes (each node has 2 vCPUs and 7GB memory). It is ensured that there is only one OSM controller pod running with default resource quota during the test. OSM is configured in non-permissive traffic mode.
+This test is performed on a Kubernetes cluster provisioned and managed by [Microsoft Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service/). The cluster has 100 Standard_DS2_v2 nodes (each node has 2 vCPUs and 7GB memory) and 3 Standard_D32s_v3 nodes (32 vCPUs and 128GB memory) reserved for high intensity tests. It is ensured that there is only one OSM controller pod running with default resource quota during the test. OSM is configured in non-permissive traffic mode.
```

Comment:
what does 3 nodes reserved for high intensity tests mean?

Reply:
This special high intensity nodepool was created and only has 3 nodes due to my resource quota of the cluster.

I've just upscaled my cluster. We will re-run the tests with larger nodes and cluster.